x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/job/results/ModelPlotTests.java

```diff
@@ -30,7 +30,7 @@ protected ModelPlot createTestInstance() {
 
     public ModelPlot createTestInstance(String jobId) {
         ModelPlot modelPlot =
-                new ModelPlot(jobId, new Date(randomLong()), randomNonNegativeLong(), randomInt());
+                new ModelPlot(jobId, new Date(randomLongBetween(0, 3000000000000L)), randomNonNegativeLong(), randomInt());
```

Comment:
Would it make sense to also introduce a `#randomDate()` method similarly to `AutodetectResultProcessorIT`?